src/index.html

```diff
@@ -11,12 +11,94 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <img
+          src="images/logo.png"
+          alt="Moyo logo"
+        />
+      </div>
```

Comment:
The logo should be link 
<img width="548" height="53" alt="image" src="https://github.com/user-attachments/assets/64b3e7d2-e837-4d57-a1e2-ea5a420ab7fd" />